tools/CreateMappings_rules.json

```diff
@@ -37,6 +37,10 @@
     "group": "Virtual Machines",
     "alias": "VM Snapshots"
   },
+  {
+    "regex": "Orbital",
```

Comment:
Use "module" instead of "regex"

Reply:
Should all "regex" in this file be replaced with "module"?

Reply:
Please just update orbital

Reply:
Ok, done.